### PR TITLE
Added replacement for lodash methods under ignitus-UtilityFunctions

### LINKED ISSUE #399: Bundle size optimisation.
We are relying on may external modules, which in the end increase the bundle size.
for instance - Imagine a use case of using **lodash**, like in order to _concat 2 arrays_ we can make use of fn offered by **lodash** know as` _.concat(array, [values])`, but if we import **lodash** just for doing this, it doesn't make sense and will can increase the **downloading** time of the application by some `ms`.

> Task: 

Look for such instances for different **npm** packages in the codebase, it can also be some animation/transition that can be achieved by making use of **css**, instead of importing an **npm** package.
Cost of adding any **npm** package can be seen here, https://bundlephobia.com/

You can find list of all `lodash` methods here as well, https://github.com/lodash/lodash

### PR DESCRIPTION
**Pull request by abhu-A-J**
fixes #399 

Made replacements for the lodash methods and kept them under ignitus-UtilityFunctions.
The functions dose'nt cover all the edge cases as lodash methods since they didn't seem a necessity based on how you were using the method. However the methods will work same way you would expect them to work  based on your inputs on the codebase.

### CODE REVIEW

File: src/ignitus-UtilityFunctions/utilityFunctions.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+function isEqual(value, other) {
+  if (value === other) return true;
+  return JSON.stringify(value) === JSON.stringify(other);
+}
+
+
+export default isEmpty;
+export default isObject;
```
[the remark this answers is not in the document]
Comment (debck) [reply src/ignitus-UtilityFunctions/utilityFunctions.js]: We can add all exports in one line `export default {foo, bar, baz}
`

**Comment by abhu-A-J** [comment]
> And please fix DeepScan issues.

@divyanshu-rawat  I'm done with resolving these issues. 

Now based on the examples given on the documentation of [lodash](https://lodash.com/docs/4.17.15)  it's failing only one test case for ` isEmpty(1)` and equivalent and that's since I didn't really consider this case because I don't see it being in use. For others you can check my [repl](https://repl.it/@abhuAJ/VariableImpoliteEllipses)  where I have shown in it detail. Let me know if you want any more changes

**Comment by divyanshu-rawat** [comment]
@abhu-A-J yes sounds good to me, but what I meant is can you also replace them with the `lodash` functions used in the app for instance here e.g - https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-Authentication/ignitus-StudentLogin/actions.js#L2, and test whether after replacement no new bug is introduced.

**Comment by abhu-A-J** [comment]
> @abhu-A-J yes sounds good to me, but what I meant is can you also replace them with the `lodash` functions used in the app for instance here e.g - https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-Authentication/ignitus-StudentLogin/actions.js#L2, and test whether after replacement no new bug is introduced.

@divyanshu-rawat They seem to work just fine as before, I replaced them and tried no new bug was introduced as far as I can see. Definitely let me know if something concerns you, I'll address them as well.

**Comment by divyanshu-rawat** [comment]
> > @abhu-A-J yes sounds good to me, but what I meant is can you also replace them with the `lodash` functions used in the app for instance here e.g - https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-Authentication/ignitus-StudentLogin/actions.js#L2, and test whether after replacement no new bug is introduced.
> 
> @divyanshu-rawat They seem to work just fine as before, I replaced them and tried no new bug was introduced as far as I can see. Definitely let me know if something concerns you, I'll address them as well.

Thanks just merged it 😅

**Comment by abhu-A-J** [comment]
> > > @abhu-A-J yes sounds good to me, but what I meant is can you also replace them with the `lodash` functions used in the app for instance here e.g - https://github.com/Ignitus/Ignitus-client/blob/master/src/ignitus-Authentication/ignitus-StudentLogin/actions.js#L2, and test whether after replacement no new bug is introduced.
> > 
> > 
> > @divyanshu-rawat They seem to work just fine as before, I replaced them and tried no new bug was introduced as far as I can see. Definitely let me know if something concerns you, I'll address them as well.
> 
> Thanks just merged it 😅

Cool!

**Pull request merged**